app/controllers/games_controller.rb

```diff
@@ -1,2 +1,20 @@
 class GamesController < ApplicationController
+
+  def new
+    @game = Game.new
+  end
+
+  def create
+    @game = Game.new(game_params)
+    # @game.user_id = current_user.id
+    if game.save
+      redirect_to game_path(@game)
+    else
+      render :new
+    end
+  end
+
```

Comment:
Please add `private` here

Reply:
Good spot, will add that later on tonight.